Userland/Libraries/LibWeb/CSS/FontFaceSet.h

```diff
@@ -50,8 +50,12 @@ class FontFaceSet final : public DOM::EventTarget {
     virtual void visit_edges(Cell::Visitor&) override;
 
     JS::NonnullGCPtr<JS::Set> m_set_entries;
-    JS::GCPtr<WebIDL::Promise> m_ready_promise; // [[ReadyPromise]]
-
+    JS::GCPtr<WebIDL::Promise> m_ready_promise;            // [[ReadyPromise]]
+                                                           //
+    Vector<JS::NonnullGCPtr<FontFace>> m_loading_fonts {}; // [[LoadingFonts]]
+    Vector<JS::NonnullGCPtr<FontFace>> m_loaded_fonts {};  // [[LoadedFonts]]
+    Vector<JS::NonnullGCPtr<FontFace>> m_failed_fonts {};  // [[FailedFonts]]
+                                                           //
```

Comment:
You need to visit() these in `visit_edges()`, which is how the GC finds what's being referenced. Should just be a case of `visitor.visit(m_loading_fonts)` etc.

Also this trailing line with a blank comment is a bit silly. :^)

Reply:
Ah okay, sorry still trying to get a solid understanding of the required actions when doing these implementations.

Agreed! my silly editor inserts them when there is a comment above the line I'm editing and I missed it. I'll remove.

Reply:
done :)